File: java/Level 1/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] arr = s.split("");
        int cnt = 0;
        
        for (int i=0;i<arr.length;i++){
            if(arr[i].equals(" ")){ // 문자열 비교
                cnt = 0;
                answer += " ";
                continue;
            }
            if(cnt%2==0)
                answer += arr[i].toUpperCase(); // char형은 Character.toUpperCase(charvar)
            else
                answer += arr[i].toLowerCase();
            cnt += 1;
        }
        
        return answer;
    }
}

File: python/Level 1/이상한 문자 만들기.py
def solution(s):
    answer = ''
    i = 0
    
    for ch in s:
        if ch == ' ':
            answer += ch
            i = 0
            continue
        
        if i % 2 == 0:
            answer += ch.upper()
        else:
            answer += ch.lower()
        i += 1
    
    return answer